src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -99,15 +100,26 @@ public static Email parseEmail(String email) throws ParseException {
 
     /**
      * Parses a {@code String payment} into an {@code Payment}.
-     * Leading and trailing whitespaces will be trimmed
+     * Leading and trailing whitespaces will be trimmed, all characters are converted to lower case
      */
     public static Payment parsePayment(String payment) throws ParseException {
         String trimmedPayment = payment.trim().toLowerCase();
         if (!Payment.isValidPayment(trimmedPayment)) {
             throw new ParseException(Payment.MESSAGE_CONSTRAINTS);
         }
-        Boolean b = Boolean.parseBoolean(trimmedPayment);
-        return new Payment(b);
+        return new Payment(trimmedPayment);
+    }
+
+    /**
+     * Parses a {@code String fee} into an {@code Fees}.
+     * Leading and trailing whitespaces will be trimmed, all characters are converted to lower case
+     */
+    public static Fees parseFees(String payment) throws ParseException {
+        String trimmedFees = payment.trim().toLowerCase();
```

Comment:
Yep this toLowerCase() as well, maybe we should remove it to keep it clear? Or is it actually required?